_data/versions.yml

```diff
@@ -1,27 +1,27 @@
-- title: v3.13.0
+- title: v3.13.1
```

Comment:
You need to _add_ v3.13.1 to this list, not replace v3.13.0.

Reply:
but why tho